Untitled.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
why is there a jupyter notebook? add it to git ignore

Reply:
DONE